packages/flex-dev-utils/src/errors/ValidationError.ts

```diff
@@ -0,0 +1,4 @@
+import { FlexPluginError } from './index';
+
+export default class ValidationError extends FlexPluginError {
```

Comment:
is this just so the code is clearer?

Reply:
Ya, this is so that we can converge all of error handling at https://github.com/twilio/flex-plugin-builder/pull/195/files#diff-f43e74d6e7453b908e4a68e270eac131R13. We'll create a mapper and use the class of the error to show the error later.